docs/templating.md

```diff
@@ -26,3 +26,13 @@ Date can be used to get information about the current date and time. HEMTT uses
 ```
 {{date \"%y%m%d\"}}
 ```
+
+## git
+Git helper can be used to get information about the git repository.
+
+Parameters:
+- `id <number>`: id (SHA-1) of HEAD revision _(number from 1 to 40, default: 8)_
```

Comment:
It could just be that I'm an idiot, but this to me reads as if you're going to give me the SHA of `HEAD~8`. I'm guessing it's actually truncated, so I'd just say `... HEAD truncated to _..._ characters`.

Reply:
```suggestion
- `id <number>`: id (SHA-1) of HEAD revision truncated to `<number>` characters _(from 1 to 40, default: 8)_
```